src/ripple/app/misc/NetworkOPs.cpp

```diff
@@ -1000,6 +984,37 @@ NetworkOPsImp::setAccountHistoryJobTimer(SubAccountHistoryInfoWeak subInfo)
         [this, subInfo]() { setAccountHistoryJobTimer(subInfo); });
 }
 
+void
+NetworkOPsImp::setBatchApplyTimer()
+{
+    using namespace std::chrono_literals;
+    static auto const batchInterval = 100ms;
```

Comment:
_nitpick:_ Even better:  `auto constexpr batchInterval = 100ms;`

The code generation will be identical.  But the reader doesn't have to know that the compiler is optimizing away the "once flag" that usually hides under the hood of function local statics.

Reply:
fixed